pkg/util/util.go

```diff
@@ -1141,8 +1141,15 @@ func ValidateURL(sourceURL string) error {
 	if err != nil {
 		return err
 	}
-
-	if len(u.Host) == 0 || len(u.Scheme) == 0 {
+	scheme := u.Scheme
```

Comment:
@GeekArthur can we use  `url.ParseRequestURI` in place of `url.Parse`?  This seems to check for the validation we are using
https://golang.org/pkg/net/url/#ParseRequestURI`

Reply:
Yes, we can switch to use that as it already covers the scheme validation, but we still need add the host validation by ourselves.